cdap-docs/vars

```diff
@@ -19,5 +19,4 @@ WITH_JAVADOCS="$WITHOUT"
 MANUALS="introduction developers-manual cdap-apps admin-manual integrations examples-manual reference-manual faqs"
 
 # This needs to be explicitly set as Git has trouble identifying it.
-GIT_BRANCH_TYPE="develop"
-GIT_BRANCH_PARENT="develop"
+GIT_BRANCH_PARENT="release/3.3"
```

Comment:
It needs to be the parent branch; once that is set, it doesn't change for that line. Each time you cut a new release branch is when it happens. Unfortunately, I've found no other way of doing this...